app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,279 @@
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      dummy_client.common_followers(self, other)
+    end
+
+    def removing
+      return [] unless TwitterUser.has_more_than_two_records?(uid, user_id)
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        dummy_client.removing(old_one, new_one)
+      end.flatten
+    end
+
+    def removed
+      return [] unless TwitterUser.has_more_than_two_records?(uid, user_id)
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        dummy_client.removed(old_one, new_one)
+      end.flatten
+    end
+
+    def replying(options = {})
+      if statuses.any?
+        client.replying(statuses.to_a, options)
+      else
+        client.replying(uid.to_i, options)
+      end.map { |u| u.uid = u.id; u }
+    end
+
+    def replied(options = {})
+      result =
+        if ego_surfing?
+          if mentions.any?
+            mentions.map { |m| m.user }.map { |u| u.uid = u.id; u }
+          else
+            client.replied(uid.to_i, options)
+          end
+        elsif search_results.any?
+          uids = dummy_client._extract_uids(search_results.to_a)
+          dummy_client._extract_users(search_results.to_a, uids)
+        else
+          client.replied(uid.to_i, options)
+        end.map { |u| u.uid = u.id; u }
+
+      (options.has_key?(:uniq) && !options[:uniq]) ? result : result.uniq { |u| u.uid.to_i }
+    end
+
+    def favoriting(options = {})
+      if favorites.any?
+        client.favoriting(favorites.to_a, options)
+      else
+        client.favoriting(uid.to_i, options)
+      end.map { |u| u.uid = u.id; u }
+    end
+
+    def inactive_friends
+      dummy_client._extract_inactive_users(friends, authorized: ego_surfing?)
+    end
+
+    def inactive_followers
+      dummy_client._extract_inactive_users(followers, authorized: ego_surfing?)
+    end
+
+    def clusters_belong_to
+      text = statuses.map{|s| s.text }.join(' ')
+      dummy_client.clusters_belong_to(text)
+    end
+
+    def close_friends(options = {})
+      min = options.has_key?(:min) ? options.delete(:min) : 1
+      options.merge!(uniq: false, min: min)
+      user = Hashie::Mash.new({
+                                replying: replying(options),
+                                replied: replied(options),
+                                favoriting: favoriting(options)
+
+                              })
+
+      client.close_friends(user, options).map { |u| u.uid = u.id; u }
+    end
+
+    def inactive_friends_graph
+      inactive_friends_size = inactive_friends.size
+      friends_size = friends_count
+      [
+        {name: I18n.t('legend.inactive_friends'), y: (inactive_friends_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.not_inactive_friends'), y: ((friends_size - inactive_friends_size).to_f / friends_size * 100)}
```

Comment:
__[rubocop]__
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [124/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)